console/console-init/ui/src/Pages/AddressSpaceDetail/AddressList/AddressListPage.tsx

```diff
@@ -121,7 +121,7 @@ export const AddressListPage: React.FunctionComponent<IAddressListPageProps> = (
             Namespace: addressBeingEdited.namespace
           },
           jsonPatch:
-            '[{"op":"replace","path":"/Plan","value":"' +
+            '[{"op":"replace","path":"/spec/plan","value":"' +
```

Comment:
In light of the discussion, shouldn't this be /Spec/Plan

Reply:
The path variable is simply passed through to the address client interface.  Therefore, it needs to be lowercase for the server to be able to evaluate it on the resource.
We can accept title case, if we want to be more consistent.  But we will need to add a translation in the resolver to make it lower case.  A simple string replace function would be error prone.
I think it is best to leave this matching the openshift objects.  But am willing to do the translation for consistency purposes.

Reply:
My review comment is spurious.   I retract it.